File: LeetCode/412.c
/**
 * Return an array of size *returnSize.
 * Note: The returned array must be malloced, assume caller calls free().
 */
char** fizzBuzz(int n, int* returnSize) {
    char** list = malloc(sizeof(char*) * n);
    for(int i=0; i<n; i++) {
        list[i] = malloc(sizeof(char) * 10);
        list[i][0] = '\0';
        if(!((i+1)%15)) strcpy(list[i], "FizzBuzz");
        else if(!((i+1)%5)) strcpy(list[i], "Buzz");
        else if(!((i+1)%3)) strcpy(list[i], "Fizz");
        else sprintf(list[i], "%d", i+1);
    }
    *returnSize = n;
    return list;
}

File: LeetCode/412.py
class Solution:
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        fin = []
        for num in range(1, n+1):
            fizz = not num%3
            buzz = not num%5
            ret_str = ""
            
            ret_str += "Fizz" if fizz else ""
            ret_str += "Buzz" if buzz else ""
            ret_str += str(num) if not len(ret_str) else ""
            fin.append(ret_str)
        return fin
